sentencepiece/apt/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```

Comment:
Did you copy this file from the Apache Arrow repository?
Then we should use Apache License 2.0 for this.